flake.nix

```diff
@@ -50,9 +50,14 @@
           rustc
           cargo
           rustfmt
+
+          # Fix the error while building the project:
+          # `ld: library not found for -liconv`
+          libiconv
         ];
 
-        RUST_BACKTRACE = 1;
+        # WARNING: By enabling this the integration tests will fail
+        # RUST_BACKTRACE = 1;
```

Comment:
Would there be a better solution for this? Keeping backtrace on in the devshell would be ideal. Adding this to `tests/exec/nonexistent.unix.toml` fixes the issue:

```toml
[env.add]
RUST_BACKTRACE = "0"
```